### PR TITLE
[CI test] Qt4WebKit masking

### PR DESCRIPTION
**Pull request by a17r**
https://bugs.gentoo.org/620684

### CODE REVIEW

File: profiles/base/package.use.mask [hunk 1]
```diff
@@ -7,6 +7,17 @@
...
 
+# Andreas Sturmlechner <asturm@gentoo.org> (14 Oct 2017)
+# Depends on masked dev-qt/qtwebkit:4
+dev-qt/assistant:4 webkit
+dev-qt/designer:4 webkit
+dev-qt/qt-meta:4 webkit
+dev-qt/qtdeclarative:4 webkit
+dev-qt/qtdemo:4 webkit
+dev-python/PyQt4 webkit
+dev-python/pyside webkit
```
Comment (Pesa) [inline profiles/base/package.use.mask]: can you add a `:0` slot here, to avoid affecting `pyside:2` in qt overlay

**Comment by a17r** [comment]
@Pesa CI is green, so I'll merge whenever you ACK

**Comment by markwright** [comment]
I ACK dropping all the old rstudio versions, thanks!

**Comment by Arfrever** [comment]
I think that stabilization of new versions of Fcitx packages will occur in next week.

In `<app-i18n/fcitx-libpinyin-0.5.0`, dependency on `dev-qt/qtwebkit:4` is conditional on `dictionary-manager` USE flag, so adding `<app-i18n/fcitx-libpinyin-0.5.0` to `profiles/package.mask` would be incorrect anyway.

If you cannot wait, then add `<app-i18n/fcitx-libpinyin-0.5.0 dictionary-manager` to `profiles/base/package.use.mask`.

**Comment by a17r** [comment]
And that's it. Thanks all!

**Pull request closed without merging**